### DISCUSSION TITLE
Do the string status indicators mean anything

### DISCUSSION THREAD

**Discussion by comradekingu** [WeblateOrg/weblate#4698] [Q&A]
Flying a bit low on not fully thought through suggestions for the Zen editor with #4696 and #4697 
The status field doesn't fit with that, and keeping the flow of information in central areas and consistent questions the status field.

**Does anyone have any use of the status section in the top left?**
![bilde](https://user-images.githubusercontent.com/13802408/96053804-c9849480-0e6f-11eb-96eb-88e068f9cd18.png)
![bilde](https://user-images.githubusercontent.com/13802408/96052510-419d8b00-0e6d-11eb-9d17-9d32e50721ce.png) means **untranslated** (this status is usually green, but that is the soft-keyboard)
I don't see why that is relevant. In Zen mode it just means the editing started from somewhere. There is no way to not save edits other than to Ctrl+Z all the way back
![bilde](https://user-images.githubusercontent.com/13802408/96052695-98a36000-0e6d-11eb-873a-64804f2caffa.png) means **translated** (this is usually green, but so is the soft keyboard, so that isn't very distinguished)
(Why is the soft-keyboard green is the bigger question.)
The statuses don't update on the fly (?) while one is in the editor, so why doesn't the user already know that the string is being edited?
![bilde](https://user-images.githubusercontent.com/13802408/96053225-a7d6dd80-0e6e-11eb-859d-672a4f618d79.png) means some sort of **error** (I think surrounding the editing field or ideally where the error is would be better)
![bilde](https://user-images.githubusercontent.com/13802408/96053249-b4f3cc80-0e6e-11eb-8c4d-7f5ab4e8606e.png) means the string is **commented** (the functionality show them is not good)
I had an idea for showing comments in Zen in #2767
There are probably other statuses (will update when I see them), but for XML errors, fuzzy and whatnot, I think the colours from the bars should be used. Red and green are problematic for colour-blind users.
There are palettes that work a lot better, but then one that signifies "good" "bad" and so on is needed.
Grayscale where reviewed is white, and then one "look here, error" in colour could do?

Semi-related: #4476 

My current assumptions:
* The soft-keyboard can be not-green, meaning regular text colour
* The "translated" can be removed, and if not shouldn't be green because colour draws attention.

![bilde](https://user-images.githubusercontent.com/13802408/96067556-86cfb600-0e89-11eb-9fca-a7a4f7bb68df.png) means "**needs editing**". However I didn't know that, because there the connection between clicking the checkbox to do so, seeing it as a string status, the colour, and finding it with the regular expression picker/search.

**Comment by nijel** [comment]
The full list of possible displayed states can be seen here: https://github.com/WeblateOrg/weblate/blob/master/weblate/templates/snippets/unit-state.html

Also, these are not used only in the zen mode, but whenever listing string - search results, nearby strings, other languages, ....

**Comment by comradekingu** [comment]
@nijel I don't understand how the "untranslated" one does anything. If there is nothing in the field, there is nothing.
If a translation exists, there is already a checkmark for it.
It could mean someone actively saved an empty translation?

Furthermore, the untranslated icon has no resemblance or connotation to "untranslated" as a component category, (which is a red squircle).
Is the untranslated icon used somewhere else and I am meant to learn what the icon means to recognize it somewhere else?
Someone deleted a translation in the history?

What would be lost if it was removed?

**Reply by nijel** [reply]
The status indication should be changed and consolidated for the 4.6 release. You are right that current status is far from being good.

**Comment by comradekingu** [comment]
![bilde](https://user-images.githubusercontent.com/13802408/108438480-a0c24080-724f-11eb-814e-9d9a797c358d.png)

This thing for suggestions is strange. Should be a questionmark instead. (?).
